cogs5e/initTracker.py

```diff
@@ -399,12 +402,18 @@ async def nextInit(self, ctx):
             await Stats.increase_stat(ctx, "rounds_init_tracked_life")
 
         out.append(combat.get_turn_str())
-
-        for co in toRemove:
+        removed = []
+        for co in to_remove:
             combat.remove_combatant(co)
-            out.append("{} automatically removed from combat.\n".format(co.name))
+            removed.append("{} automatically removed from combat.\n".format(co.name))
```

Comment:
Technically this is a bug that existed before, but adding a `\n` here causes an extra line break, since it joins with `\n` later